spec/helpers/user_helper_spec.rb

```diff
@@ -61,4 +61,65 @@
       end
     end
   end
+
+  describe "display_pending_email_change_banner?" do
+    context "when the user doesn't have an unconfirmed email" do
+      let(:user) { FactoryBot.create(:user, :data_provider, unconfirmed_email: nil) }
+
+      it "does not display pending email change banner" do
+        expect(display_pending_email_change_banner?(user)).to be false
+      end
+    end
+
+    context "when the user doesn't has the same unconfirmed email as current email" do
```

Comment:
You've got a stray "doesn't" in this context name, and in the context below.

Reply:
That's what I get for copying my test cases.. Thank you!